### ISSUE TITLE
Broke afterward build beckup 

### ISSUE THREAD

**Issue by tomtomclub** [Tikipeter/repository.tikipeter#287]
After beckup.  Reinstalling fen broke  I hoop you have something on the log 
https://paste.kodi.tv/kivozocidu

**Comment by Tikipeter** [comment]
What version of Fen are you running?

**Comment by tomtomclub** [comment]
> What version of Fen are you running?

The last one. 1.8.13  The older fen is fine 
No a problem

**Comment by Tikipeter** [comment]
Did you reinstall?

**Comment by tomtomclub** [comment]
Yes the only way is reinstalling and try it but the same results after backup build . Broke

**Comment by Tikipeter** [comment]
**How are you doing the backup?**

Something is going wrong with it, and it is not including the `cache` folder inside Fen (it might not be including others as well, but I can't know that because of the error for the `cache` folder happening first).

The `cache` folder in Fen is fairly new, I moved a couple of modules into it one or two updates ago. Perhaps the program you are using to backup is using a cached instance of Fen, or at least a cached instance of what it thinks Fen's folder structure will be.

**Comment by tomtomclub** [comment]
Hmm sorry I told you venom en more addon no problems. I use openwizard.  Do you have sulision? It's hard solved  the issue

**Comment by Tikipeter** [comment]
I don't have a solution I'm afraid. The zip file for installation includes all the correct files and folders. That is all I have control over.

I have no idea how openwizard makes the backup, but something is not right there. An addon needs to be able to add files/folders to their structure when required, without a backup program breaking because of it.

I don't think there is anything I can do here to help.

Maybe I'll install openwizard and see what it's doing and why it's failing.

**Comment by Tikipeter** [comment]
I downloaded openwizard and think I see the problem.

When openwizard is making the backup, it has a list of directories to ignore. `cache` is in that list. So it is ignoring Fen's `cache` folder when making the backup.

I will rename this `cache` folder for next update so that openwizard will work again for you.

**Comment by tomtomclub** [comment]
Thanks you so much ! For the help and time 
🍺🍺

**Comment by Tikipeter** [comment]
No worries. Let me know after the update if it fixed the issue.
